webpack.config.dist.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-extraneous-dependencies, no-console */
+
+const css = require('./css.config');
+const path = require('path');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
Maybe transition to `MiniCssExtractPlugin` if possible?

Reply:
This is using webpack 2 (which I just upped to 3)

I dont want to make too many changes just yet

Reply:
Fair enough.